### PR TITLE
feat: ds4ch hero component & style

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

**Review by rwd — CHANGES_REQUESTED** [review]

File: packages/portal/src/lang/en.js [hunk 1]
```diff
@@ -292,6 +293,7 @@ export default {
       "title": "We can't seem to find what you are looking for."
     }
   },
+  "europeana": "Europeana",
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/lang/en.js]: that's removed.

File: packages/portal/src/components/DS4CH/DS4CHLandingHero.vue [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+          class="btn btn-primary d-inline-flex align-items-center mt-1 mt-md-4 mb-0"
+        >
+          {{ cta.text }}
+        </SmartLink>
+      </header>
+      <div
+        id="europeana-logo"
+        class="mt-auto"
+      >
+        <h1>
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/components/DS4CH/DS4CHLandingHero.vue]: That's now a `p`

File: packages/portal/src/components/DS4CH/DS4CHLandingHero.vue [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+          {{ $t('ds4ch.broughtBy') }}
+        </h1>
+        <img
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/components/DS4CH/DS4CHLandingHero.vue]: `img` moved inside the `p` and added styles for margins. Besides updated the responsive font and logo sizes to match the design.

File: packages/portal/src/components/DS4CH/DS4CHLandingHero.vue [hunk 4]
```diff
@@ -0,0 +1,348 @@
+<template>
+  <div class="landing-hero">
+    <div
+      class="hero-background responsive-backround-image"
+      :style="imageCSSVars"
+      role="img"
+      :aria-label="heroImage.image.description"
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/components/DS4CH/DS4CHLandingHero.vue]: Done.

**Pull request merged**